configure.ac

```diff
@@ -753,6 +753,9 @@ if test x$use_lcov_branch != xno; then
   AC_SUBST(LCOV_OPTS, "$LCOV_OPTS --rc lcov_branch_coverage=1")
 fi
 
+dnl Check for __int128
+AC_CHECK_TYPES([__int128])
```

Comment:
Thanks, I have split up the large commit into four separate ones and melted the other unit test commit with one of them. I think this will help with reviews.